tests/blob/test_blob_spec.lua

```diff
@@ -78,6 +78,7 @@ describe(" Blob Methods Tests ", function()
 		local readme_lookup_prefix = lib.luagit2_blob_lookup_prefix(repo,README_blob_oid,5)
 		local readme_lookup_prefix_oid = lib.luagit2_blob_id(readme_lookup_prefix)
 		assert.are.equal(README_oid_string,lib.luagit2_oid_tostr_s(readme_lookup_prefix_oid))
+		lib.luagit2_blob_free(readme_lookup_prefix)
```

Comment:
Some feedback, this code modification doesnt have anything to do with the tree module, so it should have been in the blob PR. Remember, in the future, to make PRs be about a particular module, feature, bug fix, without adding a mix of things that dont have much to do with each other together :)

Reply:
Yes mam, For all the active PRs newer to #24 I have the followed similar thoughts. all code + tests in one pr only.